### PR TITLE
Clustered graph serialization

### PR DESCRIPTION
**Pull request by eartser**

### CODE REVIEW

File: code-submissions-clustering-core/src/test/kotlin/org/jetbrains/research/code/submissions/clustering/SerializationTest.kt [hunk 1]
```diff
@@ -22,6 +24,34 @@ class SerializationTest : ParametrizedBaseWithUnifierTest(getTmpProjectDir()) {
         }
     }
 
+    @ParameterizedTest
+    @MethodSource("getTestData")
+    fun testSerializeClusteredGraph(dataFrame: DataFrame<*>) {
+        WriteCommandAction.runWriteCommandAction(mockProject) {
+            val graph = dataFrame.loadGraph(mockContext)
+            val clusterer = SubmissionsGraphHAC(3.0)
```
Comment (nbirillo) [inline code-submissions-clustering-core/src/test/kotlin/org/jetbrains/research/code/submissions/clustering/SerializationTest.kt:32]: Please, add a const for 3

**Review by nbirillo — CHANGES_REQUESTED** [review]
Great!

File: code-submissions-clustering-core/src/main/kotlin/org/jetbrains/research/code/submissions/clustering/util/SerializationUtil.kt [hunk 2]
```diff
@@ -56,6 +64,46 @@ fun ProtoSubmissionsGraph.toGraph(): SubmissionsGraph {
     return SubmissionsGraph(graph)
 }
 
+// COPY
```
[the remark this answers is not in the document]
Comment (eartser) [reply code-submissions-clustering-core/src/main/kotlin/org/jetbrains/research/code/submissions/clustering/util/SerializationUtil.kt]: Oops, useless comment indeed, removed it.

**Review by nbirillo — APPROVED** [review]
Great!

**Pull request merged**